### DISCUSSION TITLE
What is the meaning and function of this keySignature

### DISCUSSION THREAD

**Discussion by hhly** [CoderLine/alphaTab#1065] [Q&A] [answered]
<img width="308" alt="image" src="https://user-images.githubusercontent.com/29226405/204447202-c593d72f-d859-48db-abda-6f26e93b829d.png">
What is the meaning and function of this keySignature

**Comment by Danielku15** [comment, marked as answer]
This is the [`alphaTab.model.KeySignature`](
https://github.com/CoderLine/alphaTab/blob/develop/src/model/KeySignature.ts) enumeration. The numeric value represents for positive numbers the number of sharp accidentals and for negative the number of flats. But I recommend not to rely on any numeric values for the meaning but the enumeration with a proper name.
